orbs/pulumi.yml

```diff
@@ -42,6 +42,16 @@ commands:
       - run: 
           name: "Install Pulumi CLI, if needed"
           command: |
+              # Check if Pulumi CLI is already installed with specified version
+              if command -v pulumi &> /dev/null; then
+                  INSTALLED_VERSION=$(pulumi version | cut -d 'v' -f2)
+                  if [ "<< parameters.version >>" == "latest" ] || [ "$INSTALLED_VERSION" == "<< parameters.version >>" ]; then
```

Comment:
If `parameters.version == "latest"`, I think we need to check whether the installed version is actually the latest before skipping the installation.  The installed version might be an older one otherwise, no?

To do that I think the simplest thing to do would be to check against https://www.pulumi.com/latest-version